### PR TITLE
Commits 481-531, for some reason 266-275 also show up

### PR DESCRIPTION
**Pull request by slmarii**

### CODE REVIEW

File: analysis/PX4-Autopilot.md [hunk 1]
```diff
@@ -7896,11 +7910,11 @@ We can afford a slower performance in this parsers with python2 to
 keep compability with python3.
 http://sebastianraschka.com/Articles/2014_python_2_3_key_diff.html#xrange
 ### Antipattern Category
-
+X
 ### Keyword
 performance
 ### Note
-
+This commit does not change any performance-related features.
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:7917]: I would argue there are some performance related changes made, just not of interest for this current study. Maybe change the note slightly?

File: analysis/PX4-Autopilot.md [hunk 2]
```diff
@@ -8383,11 +8397,11 @@ px4nucleoF767ZI-v1 nsh: increase CONFIG_NFILE_DESCRIPTORS from 53 to 54
    This is needed when logger is logging to file and ulog streaming gets
    activated
 ### Antipattern Category
-
+New:Hard-coded-fine-tuning
 ### Keyword
 increase
 ### Note
-
+Value changed from 51 to 54.
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:8404]: The code changes are:
```
- CONFIG_NFILE_DESCRIPTORS=47
+ CONFIG_NFILE_DESCRIPTORS=54
```
Not 51 -> 54 as is written in the note.

Interesting though, the commit message is also incorrect (done by the developer).

File: analysis/PX4-Autopilot.md [hunk 3]
```diff
@@ -8434,11 +8448,11 @@ px4-same70xplained-v1 nsh: increase CONFIG_NFILE_DESCRIPTORS from 53 to 54
...
```
Comment (IvDinten) [inline analysis/PX4-Autopilot.md:8455]: The code changes are:

```
- CONFIG_NFILE_DESCRIPTORS=53
+ CONFIG_NFILE_DESCRIPTORS=54
```

Not 51 -> 54 as is written in the note.

**Review by IvDinten — CHANGES_REQUESTED** [review]
Check comments for change requests.

File: analysis/PX4-Autopilot.md (hunk 1 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:7917]: Will do

File: analysis/PX4-Autopilot.md (hunk 2 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:8404]: Will update the note

File: analysis/PX4-Autopilot.md (hunk 3 above)
Comment (slmarii) [reply analysis/PX4-Autopilot.md:8455]: Will correct it

**Review by IvDinten — APPROVED** [review]

**Pull request merged**